tests/hf/test_tapering_hf.py

```diff
@@ -625,12 +597,13 @@ def test_hf_energy(symbols, geometry, charge):
     mol = qml.hf.Molecule(symbols, geometry, charge)
     hamiltonian = qml.hf.generate_hamiltonian(mol)(geometry)
     hf_state = np.where(np.arange(len(hamiltonian.wires)) < mol.n_electrons, 1, 0)
-    generators, paulix_ops = generate_symmetries(hamiltonian, len(hamiltonian.wires))
+    generators = qml.symmetry_generators(hamiltonian)
+    paulixops = qml.paulix_ops(generators, len(hamiltonian.wires))
     paulix_sector = optimal_sector(hamiltonian, generators, mol.n_electrons)
 
-    hamiltonian_tapered = transform_hamiltonian(hamiltonian, generators, paulix_ops, paulix_sector)
-    hf_state_tapered = transform_hf(
-        generators, paulix_ops, paulix_sector, mol.n_electrons, len(hamiltonian.wires)
+    hamiltonian_tapered = qml.taper(hamiltonian, generators, paulixops, paulix_sector)
+    hf_state_tapered = taper_hf(
```

Comment:
Same as above.

Reply:
I prefer to use `qml.name` to make sure that these functions remain high-level and the user interface is always `qml.name`.

Reply:
Maybe then use `qml.taper_hf` here to be consistent throughout? But then I also wonder if it beats the purpose of importing the functions in the first place?

Reply:
The functions `taper`, `symmetry_generators`, `paulix_ops` are not qchem-specific and they will be accessible with `qml.name`. The rest of the functions will either stay in qchem or go to a future home dedicated to operator arithmetic. So, these three functions are the only ones that will be accessible with `qml.name`.